packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -228,22 +238,31 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({
   const showImageCaptureModal = useCallback(() => {
     const close = showModal('capture-photo-modal', {
       saveFile: (file: UploadedFile) => {
-        setValue('image', file);
+        if (file) {
+          const filteredFiles = allowedImageTypes.includes('.' + file.fileName.split('.').pop().toLowerCase());
```

Comment:
Do we need to do this filtering? I assume it would be handled by the `capture-photo-modal`.